explanation/security/landing.md

```diff
@@ -65,7 +65,7 @@ The following table helps you understand how data related to you or provided by
 | Anbox Stream Gateway | Dqlite | Session and management metadata, service account IDs that identify the web client |
 | Anbox Cloud dashboard | SQLite | User emails that are used for authentication |
 
-Services used by Anbox Cloud have configuration files that contain secrets. The secrets are automatically generated and managed by the respective charms or the appliance. The authentication methods used are further defined in {ref}`sec-cryptography` for managing secrets.
+Services used by Anbox Cloud have configuration files that contain secrets. The secrets are automatically generated and managed by the respective charms or the appliance. The authentication methods used for managing secrets are explained in the security topics.
```

Comment:
I just realized I've not submitted the PR to introduce the juju secrets used by charm.
I'll do that later on.